framework/utils/utils.py

```diff
@@ -391,16 +391,20 @@ def toBytes(s):
   else:
     return s
 
-def isSingleValued(val,nanOk=True):
+def isSingleValued(val,zeroDOk=True,nanOk=True):
```

Comment:
I would place the new optional in last position. So if anybody already using this method (expecting only one attribute in the call) will not see the method behavior change.

Reply:
👍